### PR TITLE
CONSOLE-2999: Switch the OCP branding logo and title to Red Hat OpenShift logo and title

### PR DESCRIPTION
**Pull request by sg00dwin**
Changes for https://issues.redhat.com/browse/CONSOLE-2999

Additional related changes to be included for the OCP login, errors, providers Oauth Templates  
https://openshift.github.io/oauth-templates/

### CODE REVIEW

**Comment by yapei** [comment]
@yanpzhan Please help test this PR, thanks!

File: frontend/public/components/masthead.jsx [hunk 1]
```diff
@@ -6,7 +6,6 @@ import { MastheadToolbar } from './masthead-toolbar';
 import { history } from './utils';
 import okdLogoImg from '../imgs/okd-logo.svg';
 import openshiftLogoImg from '../imgs/openshift-logo.svg';
-import ocpLogoImg from '../imgs/openshift-platform-logo.svg';
```
Comment (rhamilto) [inline frontend/public/components/masthead.jsx:9]: Should also delete the logo file itself since it is not longer in use.

Comment (sg00dwin) [reply frontend/public/components/masthead.jsx:9]: updated with deleted file

**Comment by rhamilto** [comment]
/lgtm

Will remove the hold once I get confirmation `master` is open for 4.11.

File: frontend/public/components/masthead.jsx [hunk 2]
```diff
@@ -19,8 +18,8 @@ export const getBrandingDetails = () => {
       productName = 'Red Hat OpenShift';
       break;
     case 'ocp':
-      logoImg = ocpLogoImg;
-      productName = 'Red Hat OpenShift Container Platform';
```
Comment (deads2k) [inline frontend/public/components/masthead.jsx:23]: this line change affected the disruption test detection.  I'm hoping others find this when they fix the test in openshift/origin master.

**Pull request merged**